docs/Milestones.md

```diff
@@ -1,10 +1,10 @@
 # Project Electron Milestones
 
-Revised April 2016
+Revised 2017
 
 ## Discovery
 
-September 2016-December 2016
```

Comment:
The "discovery" section in the year two document should really be "archival discovery and delivery" as opposed to the discovery phase of this project, so they're really two different things. Sorry for the confusion!